mission_control/test/test_mission_control_behaviors.test

```diff
@@ -0,0 +1,11 @@
+<launch>
+  <include file="$(find mission_control)/launch/mission_control.launch" />
+  <!--
+      These test verifyes the behaviors/actions:
+        -   Fixed Rudder
+        -   Go to Waypoint
+        -   Depth Heading
+        -   Attitude Servo
+  -->
+  <test test-name="test_mission_control_fixed_rudder_action" pkg="mission_control" type="test_fixed_rudder_behavioral.py" time-limit="10.0" />
+</launch>
```

Comment:
@patapufate unrelated? I think I've seen this in #89.

Reply:
Yes and No. it was created in this PR to follow the "How to Test" Section.

Reply:
I see. Why doesn't it match the comment? Is this missing tests for the other behaviors?

Reply:
yes. Removing behavior not used.

Reply:
You sure?